tests/ssl-opt.sh

```diff
@@ -134,6 +156,26 @@ requires_config_disabled() {
     fi
 }
 
+requires_config_value_at_least() {
+    NAME="$1"
+    DEF_VAL=$( grep ".*#define.*MBEDTLS_SSL_DTLS_MAX_BUFFERING" ../include/mbedtls/config.h |
+               sed 's/^.*\s\([0-9]*\)$/\1/' )
+    VAL=$( ../scripts/config.pl get $NAME || echo "$DEF_VAL" )
+    if [ "$VAL" -lt "$2" ]; then
+       SKIP_NEXT="YES"
+    fi
+}
+
+requires_config_value_at_most() {
+    NAME="$1"
+    DEF_VAL=$( grep ".*#define.*MBEDTLS_SSL_DTLS_MAX_BUFFERING" ../include/mbedtls/config.h |
```

Comment:
`MBEDTLS_SSL_DTLS_MAX_BUFFERING` is hardcoded here; not a problem for this PR as it only uses `requires_config_value_xxx()` for this configuration option, but the function claims to be generic and hence this should be changed. I addressed it as https://github.com/hanno-arm/mbedtls/commit/e604556febc8dd666f34e200b7ebc22061bfc6ce in the follow-up PR.